src/test/fuzz/connman.cpp

```diff
@@ -128,9 +128,6 @@ FUZZ_TARGET_INIT(connman, initialize_connman)
             [&] {
                 connman.SetNetworkActive(fuzzed_data_provider.ConsumeBool());
             },
-            [&] {
-                connman.SetServices(random_service, ConsumeWeakEnum(fuzzed_data_provider, ALL_SERVICE_FLAGS));
```

Comment:
48f96385c809a3e1d957538db8864ac068aca7eb

When removing this here, it would be nice to update the addrman fuzz target to use `ConsumeWeakEnum`

Reply:
Improving the addrman fuzzer seems out of scope of this PR (since it doesn't change the addrman interface).